base/applications/clipbrd/clipbrd.c

```diff
@@ -49,25 +50,22 @@ static void SaveClipboardToFile(void)
     CloseClipboard();
 }
 
-static void LoadClipboardDataFromFile(LPWSTR lpszFileName)
+static BOOL LoadClipboardDataFromFile(LPWSTR lpszFileName)
 {
-    if (MessageBoxRes(Globals.hMainWnd, Globals.hInstance,
```

Comment:
Why moving this call outside? (you need then to duplicate it many times in different parts of the code)